site/static/agones.cast

```diff
@@ -29,7 +29,7 @@
 [5.890398, "o", "\u001b[?1l\u001b>\u001b[?2004l\r\r\n"]
 [5.891151, "o", "\u001b]2;kubectl apply -f fleet.yaml\u0007\u001b]1;kubectl\u0007"]
 [5.891306, "o", "\u001b]133;C;\u0007"]
-[6.329093, "o", "fleet.stable.agones.dev/simple-udp created\r\n"]
+[6.329093, "o", "fleet.agones.dev/simple-udp created\r\n"]
```

Comment:
I'm not sure whether the feature flag things work in this file....

Reply:
this file is interpreted by javascript to render the cast.

Reply:
I wasn't sure how to fix it properly. One option is just put this into a separate PR and merge it right before we cut 0.12.0 so that the website flips is only different from the latest release for a short period of time.

Reply:
I've moved this change into https://github.com/googleforgames/agones/pull/911.

Reply:
awesome.